### DISCUSSION TITLE
blog/zarr-python-v3-update/

### DISCUSSION THREAD

**Discussion by giscus** [zarr-developers/blog#55] [General]
# blog/zarr-python-v3-update/

A status update on the development of Zarr-Python 3

https://zarr.dev/blog/zarr-python-v3-update/

**Comment by sanketverma1704** [comment]
The next Zarr-Python developers meeting is on 5/17 @ 16:00 CEST.
Check the meeting details here: https://zarr.dev/community-calls/.
